### PR TITLE
refactor: deps installation (dockerfile for custom build)

### PR DESCRIPTION
**Pull request by dpigasin**

### CODE REVIEW

File: dockerfiles/ubuntu18/build_custom/Dockerfile [hunk 1]
```diff
@@ -6,32 +6,31 @@ FROM ubuntu:18.04 AS setup_openvino
...
-    git clone --recurse-submodules --shallow-submodules https://github.com/openvinotoolkit/openvino.git --depth 1 -b ${openvino_branch}
+    git clone https://github.com/openvinotoolkit/openvino.git \
+    --recurse-submodules --shallow-submodules --depth 1 -b ${openvino_branch} /tmp/openvino
 
 WORKDIR /tmp/openvino
 RUN chmod +x install_build_dependencies.sh; \
     ./install_build_dependencies.sh
 
-RUN chmod +x install_dependencies/install_NEO_OCL_driver.sh; \
+RUN chmod +x scripts/install_dependencies/install_NEO_OCL_driver.sh; \
```
Comment (generalova-kate) [inline dockerfiles/ubuntu18/build_custom/Dockerfile:26]: add option --no_numa and version to settings

File: dockerfiles/ubuntu18/build_custom/opencv_cmake.txt [hunk 2]
```diff
@@ -22,6 +22,5 @@
 -D WITH_JASPER=OFF
 -D WITH_EIGEN=OFF
 -D WITH_LAPACK=OFF
--D WITH_GSTREAMER=OFF
```
Comment (generalova-kate) [inline dockerfiles/ubuntu18/build_custom/opencv_cmake.txt:25]: set -D WITH_TBB=OFF

File: dockerfiles/ubuntu18/build_custom/openvino_cmake.txt [hunk 3]
```diff
@@ -2,6 +2,5 @@
 
 -D CMAKE_BUILD_TYPE=Release
 -D ENABLE_PYTHON=ON
--D NGRAPH_PYTHON_BUILD_ENABLE=ON
--D PYTHON_EXECUTABLE=/usr/bin/python3
--D ENABLE_PROFILING_ITT=ON
```
Comment (generalova-kate) [inline dockerfiles/ubuntu18/build_custom/openvino_cmake.txt:7]: Why did you off ITT profiling?

Comment (dpigasin) [reply dockerfiles/ubuntu18/build_custom/openvino_cmake.txt:7]: This option should be disabled by default, in my opinion.

**Review by generalova-kate — APPROVED** [review]

**Pull request merged**